src/app/mods/dtMap/dtMap.ts

```diff
@@ -0,0 +1,57 @@
+import { ShortcutsHelper } from "../../helpers/shortcuts.helper";
+import { CustomWindow } from "../../helpers/windowHelper/customWindow";
+import { Mod } from "../mod";
+
+export class DtMap extends Mod {
+    private shortcutsHelper: ShortcutsHelper;
+    
+    private window: CustomWindow;
+    private iframe: HTMLIFrameElement;
+
+    startMod() {
+        this.params = this.settings.option.vip.general;
+
+        let dtmapCss = document.createElement('style');
+        dtmapCss.id = 'dtmapCss';
+        dtmapCss.innerHTML = `
+            #dtmap {
+                min-width: 50vw; min-height: 30vh;
+                left: 0;
+                top: 0;
+            }
+            #dtmap .windowBody {
+                padding: 0 0 4px 0 !important;
+            }
+            #dtmap-iframe {
+                border: none;
+                width: 100%;
+                height: 100%;
+            }
+        `;
+        this.wGame.document.querySelector('head').appendChild(dtmapCss);
+
+        this.shortcutsHelper = new ShortcutsHelper(this.wGame);
+        this.shortcutsHelper.bind(this.params.dtmap_shortcut, () => this.toggle() );
+
+        this.createWindow();
+    }
+
+    private createWindow() {
+        this.window = CustomWindow.createDofusWindow(this.wGame, "Dofus Touch Map", "dtmap")
+                    .makeDraggable()
+                    .makeResizable()
+                    .addFullScreenButton();
+
+        this.iframe = document.createElement("iframe");
+        this.iframe.id = "dtmap-iframe";
+        this.iframe.src = "https://www.dofus-touch-map.com";
+
+        this.window.addContent(this.iframe).hide();
+    }
+
+    private toggle() {
+        if (this.window.isVisible()) this.window.hide();
+        else this.window.show();
+    }
+    
```

Comment:
You have to implement the function `reset()`

    public reset() {
        this.window.destroy();
        this.wGame.document.getElementById('dtmapCss').remove();
        this.shortcutsHelper.unBindAll();
        super.reset();
     } 

Thanks to Xanadu